src/hooks/useGoodsItemListQuery.ts

```diff
@@ -0,0 +1,31 @@
+import { getThemesProducts } from '@apis/themes';
+import { ThemeProductsRequest } from '@/types/requestTypes';
+import { ThemeProductsResponse } from '@/types/responseTypes';
+import { QueryFunctionContext, useInfiniteQuery } from '@tanstack/react-query';
+import { AxiosError } from 'axios';
+import { useMemo } from 'react';
+
+interface useGoodsItemListQueryProps {
+  themeKey: string;
+  rowsPerPage: number;
+}
+
+export default function useGoodsItemListQuery({ themeKey, rowsPerPage }: useGoodsItemListQueryProps) {
+  const queryKey = ['theneProduct', themeKey];
```

Comment:
```suggestion
  const queryKey = ['themeProduct', themeKey];
```

오타인 거 같은데 한번 확인 부탁드릴게요.